_static/scripts/part2/memoryobject/simple_memobj.hh

```diff
@@ -90,6 +91,7 @@ class SimpleMemobj : public MemObject
         AddrRangeList getAddrRanges() const override;
 
         /**
+
```

Comment:
Need for this change?

Reply:
I guess I add this line by mistake. Sorry about that.